pkgs/development/interpreters/tcl/mk-tcl-derivation.nix

```diff
@@ -10,6 +10,7 @@
 , nativeBuildInputs ? []
 , propagatedBuildInputs ? []
 , checkInputs ? []
+, nativeCheckInputs ? []
```

Comment:
Unused. You used `nativeInstallCheckInputs` below.

Reply:
it is used line 53. The existing code for checkInputs is weird, and I mimick it. Note that putting checkInputs into installCheckInputs is not unheard of, python package transparently move the checkPhase to installCheck.